lib/ecto/adapters/postgres/connection.ex

```diff
@@ -1142,6 +1193,12 @@ if Code.ensure_loaded?(Postgrex) do
       [?", name, ?"]
     end
 
+    # TRUE, ON, or 1 to enable the option, and FALSE, OFF, or 0 to disable it
+    defp quote_boolean(nil), do: nil
+    defp quote_boolean(value) when value in [true, "TRUE", "ON", 1], do: "TRUE"
+    defp quote_boolean(value) when value in [false, "FALSE", "OFF", 0], do: "FALSE"
```

Comment:
I think in this case we can handle just Elixir's booleans: `true` and `false`.

Reply:
Ok 👍